navit/graphics.h

```diff
@@ -162,6 +162,7 @@ struct graphics_methods {
     navit_float (*get_dpi)(struct graphics_priv * gr);
     void (*draw_polygon_with_holes) (struct graphics_priv *gr, struct graphics_gc_priv *gc, struct point *p, int count,
                                      int hole_count, int* ccount, struct point **holes);
+    int(*get_has_appeared) (void); // To check view has appeared on iOS
```

Comment:
I have no reference to the vehicle in the graphics. The vehicle has to request authorization for the locating services after the view has appeared. Otherwise the request will not be shown to the user.

Reply:
Then you could send an callback from the graphics to the vehicle when the view is ready. I know something simmular is don within another plugin, but couldn't find it again fast enough

Reply:
Done.